library/extensions/src/main/java/com/epmedu/animeal/extensions/Context.kt

```diff
@@ -90,21 +84,6 @@ fun Context.getActivity(): Activity? = when (this) {
     else -> null
 }
 
-fun Context.copyText(
```

Comment:
Don't move the function, but the call to the function

Reply:
fixed